### PR TITLE
openjdk/openjdk-11-rhel7 -> ubi8/openjdk-11

### PR DESCRIPTION
**Pull request by pjgg**
- Change deployment folder from /deployments/target to /deployments

- Upgrade Quarkus version to 1.11.0.Final
- move on to ubi-openjdk-11

### CODE REVIEW

**Comment by Sgitario** [comment]
run tests

File: common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftTestExtension.java [hunk 1]
```diff
@@ -177,8 +177,7 @@ private void doBeforeAll(ExtensionContext context) throws Exception {
                 // using `tar` works around that nicely, because the tarball created with this command will have
                 // a single root directory named `target`, which the S2I builder image will unpack to `/deployments`
                 new Command("tar", "czf", "app.tar.gz", "target").runAndWait();
-                new Command("oc", "start-build", getAppMetadata(context).appName, "--from-archive=app.tar.gz", "--follow")
-                        .runAndWait();
+                new Command("oc", "start-build", getAppMetadata(context).appName, "--from-dir=target", "--follow").runAndWait();
```
Comment (mjurc) [inline common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftTestExtension.java:180]: Does this belong to this PR?

**Comment by mjurc** [comment]
Also, perhaps a more informative commit message would be better!

File: common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftTestExtension.java (hunk 1 above)
Comment (Ladicek) [reply common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftTestExtension.java:180]: I think that's required for Quarkus 1.11 support. In which case, we could remove the `tar czf app.tar.gz target` command, that's no longer necessary.

Comment (Ladicek) [reply common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftTestExtension.java:180]: And the comment above, too :-)

File: sql-db/postgresql/src/main/resources/application.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
 quarkus.openshift.expose=true
-quarkus.s2i.base-jvm-image=registry.access.redhat.com/openjdk/openjdk-11-rhel7
-
+quarkus.s2i.base-jvm-image=registry.access.redhat.com/ubi8/openjdk-11
+
```
Comment (Ladicek) [inline sql-db/postgresql/src/main/resources/application.properties:3]: Baaaad whitespace :-)

File: common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftTestExtension.java (hunk 1 above)
Comment (Ladicek) [reply common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftTestExtension.java:180]: Ah it's already in #163, so it should be removed from here.

**Review by mjurc — APPROVED** [review]
The MR now contains updates for 1.11.0, including the changed directory structure for deploment and new base image.

Approving.

It also supersedes #163 now.

**Pull request merged**